installer/charts/rhtap-infrastructure/templates/openshift-pipelines/job-tekton-chains.yaml

```diff
@@ -29,7 +29,7 @@ spec:
       restartPolicy: Never
       containers:
         - name: tekton-chains-cosign
-          image: ghcr.io/sigstore/cosign/cosign:latest
+          image: registry.redhat.io/rhtas/cosign-rhel9:1.1.1
```

Comment:
Unfortunately, I don't see a tag that we can just follow. We'll have to make sure we keep this image reference up to date for upcoming releases.